inc/fluent_bit/class-msgpackformatter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Altis Cloud Fluent Bit MsgPack formatter.
+ *
+ * @package altis/cloud
+ */
+
+namespace Altis\Cloud\Fluent_Bit;
+
+/*
+ * TODO Copies some formatter from Monolog, need to give credit
+ * This file is part of the Monolog package.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Monolog\Formatter\FluentdFormatter;
+use RuntimeException;
+
+/**
+ * Class MsgPackFormatter
+ *
+ * Serializes a log message to Fluent Bit socket protocol
+ *
+ * @author Nathaniel Schweinberg <nathaniel@humanmade.com>
+ */
+class MsgPackFormatter extends FluentdFormatter {
+
+	/**
+	 * Construct the MsgPackFormatter class.
+	 *
+	 * @param boolean $levelTag whether to append log level to Fluent Bit tag
+	 * name or not.
+	 * @return self
+	 * @throws RuntimeException Break when msgpack extension is not available
+	 */
+	public function __construct( bool $levelTag = false ) { // phpcs:ignore WordPress.NamingConventions.ValidVariableName
+		if ( ! function_exists( 'msgpack_pack' ) ) {
+			throw new RuntimeException( 'PHP\'s msgpack extension is required to use Monolog\'s MsgPackFormatter' );
+		}
+
+		// phpcs:ignore WordPress.NamingConventions.ValidVariableName
+		$this->levelTag = $levelTag;
+	}
+
+	/**
+	 * Normalizes and formats the log record using the msgpack serialization
+	 * format.
+	 *
+	 * @param array $record Monolog log record
+	 * @return string The formatted log record
+	 */
+	public function format( array $record ): string {
```

Comment:
I think this needs an extra space (although I guess we don't sniff for this)

```suggestion
	public function format( array $record ) : string {
```